### PR TITLE
Sig 2552 style detail

### PR DESCRIPTION
**Pull request by aditudorache**
This PR fixes several style issues reported in the Incident detail and LocationPreview components

### CODE REVIEW

File: src/signals/incident-management/components/RadioInput/index.js [hunk 1]
```diff
@@ -21,35 +34,33 @@ const RadioInput = ({ name, display, values }) => {
...
+              <p>
                 <Info>
                   {label}: {info}
                 </Info>
-              ) : (
-                <br />
-              )}
-            </p>
+              </p>
+            )}
```
Comment (janjaap) [inline src/signals/incident-management/components/RadioInput/index.js:60]: This change will cause the elements below the list of radio buttons to jump when there is no info to be shown

File: src/signals/incident-management/components/RadioInput/index.js [hunk 2]
```diff
@@ -2,14 +2,27 @@ import React from 'react';
...
+  width: 100%;
+  margin-bottom: ${themeSpacing(8)};
+`;
+
 const Info = styled.span`
   color: ${themeColor('tint', 'level5')};
 `;
 
+const ValueContainer = styled.div`
+  display: block;
```
[the remark this answers is not in the document]
Comment (aditudorache) [reply src/signals/incident-management/components/RadioInput/index.js:19]: Indeed! Thanks!

File: src/signals/incident-management/components/RadioInput/index.js [hunk 3]
```diff
@@ -2,14 +2,27 @@ import React from 'react';
...
+  position: relative;
```
[the remark this answers is not in the document]
Comment (aditudorache) [reply src/signals/incident-management/components/RadioInput/index.js:20]: Yes. The elements inside this container are absolute positioned.

File: src/signals/incident-management/components/RadioInput/index.js (hunk 1 above)
Comment (aditudorache) [reply src/signals/incident-management/components/RadioInput/index.js:60]: Were will the buttons jump? I have tested it and is work as expected.

File: src/components/History/index.js [hunk 4]
```diff
@@ -8,6 +8,7 @@ import { historyType } from 'shared/types';
 
 const H2 = styled(Heading)`
   ${styles.HeaderStyles} {
+    font-weight: 400;
```
[the remark this answers is not in the document]
Comment (aditudorache) [reply src/components/History/index.js:11]: Removed.

File: src/signals/incident-management/components/RadioInput/index.js (hunk 1 above)
Comment (janjaap) [reply src/signals/incident-management/components/RadioInput/index.js:59]: Likely related to SIG-2615. Probably not an issue that content below the radio buttons shifts down when a radio button is selected that has info text below it.

**Pull request merged**